apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -338,25 +344,34 @@ export function WriteInsAdjudicationScreen(): JSX.Element {
 
   const queryClient = useQueryClient();
   const apiClient = useApiClient();
-  const writeInsQuery = getWriteIns.useQuery({ contestId: contest.id });
+  const writeInQueueMetadataQuery =
+    getWriteInAdjudicationQueueMetadata.useQuery({ contestId });
```

Comment:
We're fetching this only to get the `pending` count - we could get the `total` count from the queue. It's a bit overkill to do a full table scan after every adjudication, but I realized I'm overthinking this.